Jenkinsfile

```diff
@@ -83,24 +82,14 @@ timestamps {
           }
         },
         robotest : {
-          if (params.RUN_ROBOTEST == 'run') {
+          if (env.RUN_ROBOTEST == 'run') {
             withCredentials([
-                [
-                  $class: 'UsernamePasswordMultiBinding',
```

Comment:
Hmm, I believe robotest is using these to pull static assets like prebuilt installers from s3 - is this not necessary anymore?

Reply:
Ah - just noticed https://github.com/gravitational/robotest/issues/202 that should address this - nvm.

Reply:
We currently pull everything from get.gravitational.io for the nightly and PR runs.  You're correct that if we were to use S3 artifacts, we'd need these creds again.

It sounds like we may be moving back to AWS anyhow soon enough, at which point I'll have a patch to cutover.  These just weren't used currently.